package.json

```
@@ -17,6 +17,7 @@
     "format:verify": "prettier lib/**/*.ts --check",
     "lint": "npx tslint -p tsconfig.json -c tslint.json \"lib/**/*.ts\" -e \"*.spec.ts\" --force",
     "test": "jest",
```

point this one directly to the top-level jest config

```suggestion
    "test": "jest --config jest.config.js",
```